src/app/project/board/board.component.ts

```diff
@@ -262,6 +262,22 @@ export class BoardComponent implements OnInit {
         this.keywords = this.projectMetadata['keywords'];
     }
 
+    // download metadata
+    downloadMetadata() {
+        const blob: Blob = new Blob([JSON.stringify(this.metadata)], {type: 'application/json'});
+        const fileName: string = 'metadata.json';
+        const objectUrl: string = URL.createObjectURL(blob);
+        const a: HTMLAnchorElement = document.createElement('a') as HTMLAnchorElement;
```

Comment:
You don't need to annotate type if the value of properties it's already initialised. But I'm not sure which convention we follow in DSP-APP. It's generally not bad, but redundant.

Reply:
I also don't know. May be @flavens can comment it and I will use it next time.